lua/swamp/misc/sh_titles.lua

```diff
@@ -213,3 +276,25 @@ AddTitle("", {
 --NOMINIFY
 --TODO: titles where you don't have a description but the title itself hints at what you do? like a secret achievement but not totally secret
 --TODO: if threshold=true, its 1 but you can put better text on the button in that case
+
+if SERVER then
+    local TitleFunctions =
+    {
+        ["weapon_gauntlet"] = function(atk) atk:AddStat("snaps") end,
```

Comment:
for string keys do `keyname=value` not `["keyname"]=value`

Reply:
Think I got it.

Reply:
Throwing this into gluaint, it's not happy about the key being a string and thinks that `keyname=value` has an unexpected `=`. Is this just gluaint being weird?